mirrord-macro/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+use proc_macro2::Span;
+use quote::quote;
+use syn::{Ident, ItemFn};
+
+#[proc_macro_attribute]
```

Comment:
I suggest to add a docstring that describes input -> output for clarity (I really hate proc macro syntax :))